doc/field_guide__eclipsing.md

```diff
@@ -0,0 +1,32 @@
+## Eclipsing variables (ecl)
+
+Eclipsing variables are binary stars systems in which the orbital plane lies nearly in the line of sight of the observer. The binary components pass in front of one another from observer's perspective and so their combined brightness decreases periodically.
+
+### Classification and numbers
+- Supertypes
+  - variable
+  - periodic
+- Subtypes
+  - EA
+  - EB
+  - EW
+- Occurrence rate: very common, about 10<sup>6</sup> expected in ZTF data
+
+### ZTF light curves
+![ZTF eclipsing](data/eclipsing.png)
+
+#### Description
+Eclipsing variables show regular, periodic "V-shape" dips in their phase folded light curves, which may be sharp or rounded, depending on the system. Their periods may range from minutes to years. Eclipse amplitudes vary from ~0.1 to several magnitudes.

```

Comment:
Is it just survey sensitivity that prevents us from going below ~0.1? Maybe we explicitly call that out as for ZTF then.